Dockerfile-build

```diff
@@ -0,0 +1,30 @@
+FROM mhart/alpine-node:4.8.0
+
+RUN apk update && apk upgrade && apk add git make gcc g++ linux-headers openssl python
```

Comment:
Damn, sucks that we need all these dependencies.

Could this be reduced to `apk add --no-cache git make gcc g++ linux-headers openssl python` ?